packages/contracts-bedrock/src/L2/interfaces/IEntrypoint.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/// @title IEntrypoint
+/// @notice Interface for contracts serving as entry points for the relaying process via L2ToL2CrossDomainMessenger.
+interface IEntrypoint {
```

Comment:
we wont need this interface at all now, as there is gonna be no callback with the context.